rules/autoconsent/burpee-com.json

```diff
@@ -1,6 +1,9 @@
 {
     "name": "burpee.com",
     "cosmetic": true,
+    "runContext": {
+        "urlPattern": "^https://(www\\.)?burpee.com"
+    },
```

Comment:
despite the name, it seems like this is a [common popup](https://github.com/duckduckgo/autoconsent/blob/a504f8559e3f5d63c81208c6d02902311d4adc8f/data/coverage.json#L1494). Do you think it can still be a generic rule?